### DISCUSSION TITLE
Studio crashes constantly

### DISCUSSION THREAD

**Discussion by wallyflower** [community/community#83063] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am running the latest Studio software on a 14" MacBook Pro
Using Parallels virtual machine.
it works V/G until the crash occurs.
I run it in windows11 because of the repair object feature (non manifold repair)
Has anyone else ran into this problem?

**Comment by ooye-sanket** [comment]
This might help you:
Check System Requirements: Ensure that your virtual machine and Windows 11 setup meet the system requirements for the Studio software.
Update Parallels and Windows: Make sure you have the latest updates for Parallels and Windows 11. Sometimes, software updates can address compatibility issues.
Review Crash Logs: Check for any crash logs or error messages that might provide insights into what's causing the crashes. They could be in the event viewer or logs specific to Parallels.

**Comment by codesmith-emmy** [comment]
Please check that you have allocated enough memory to your virtual machine and try again. You may also try shutting down your MacBook completely and rebooting again. I hope this helps.
